src/main/java/org/apache/sling/jcr/repoinit/impl/RepositoryInitializerFactory.java

```diff
@@ -177,5 +194,13 @@ protected void applyOperations(Session session, List<Operation> ops, String logM
         }
     }
 
+    
+    protected static boolean isDeveloperModeEnabled() {
+        String dm = System.getProperty(PROPERTY_DEVELOPER_MODE);
```

Comment:
This would make the code more compact, but largely a matter of taste.

```suggestion
        String dm = System.getProperty(PROPERTY_DEVELOPER_MODE, "");
```

Reply:
good one, thanks.